pkg/controller/nodes/nodes_controller_test.go

```diff
@@ -179,3 +105,82 @@ func TestNodesReconciler_NodeNotFound(t *testing.T) {
 	_, err := nodesCache.Get("node2")
 	assert.Equal(t, err, ErrNotFound)
 }
+
+func TestNodeReconciler_NodeHasTaint(t *testing.T) {
+	fakeClient := createDefaultFakeClientWithScheme()
+	dtc := createDTMockClient("1.2.3.4", "HOST-42")
```

Comment:
`dtc` is used here, while `dtClient` is used on other functions, if you want to be consistent. Same with `ctrl` and `reconciler`.